backend/src/main/kotlin/fr/gouv/cnsp/monitorfish/domain/use_cases/prior_notification/GetPriorNotifications.kt

```diff
@@ -59,10 +68,40 @@ class GetPriorNotifications(
 
                 finalPriorNotification
             }
-
         val priorNotifications = enrichPriorNotificationsWithReportingCount(priorNotificationsWithoutReportingsCount)
 
-        return priorNotifications.filter { !it.logbookMessageTyped.logbookMessage.isDeleted }
+        val ascendingSortedPriorNotifications = priorNotifications.sortedWith(
+            compareBy { priorNotification ->
```

Comment:
```suggestion
            when (sortDirection) {
                Sort.Direction.ASC -> compareBy {
                Sort.Direction.DESC -> compareByDescending {
```
On pourrait éviter un `reverse()` par la suite avec un `when` ici (et sortir le body du `compare` dans une fonction), c'est plus du taste car on n'a pas non plus des milliers d'éléments